src/fw/freertos_application.c

```diff
@@ -257,10 +257,22 @@ extern void vPortSuppressTicksAndSleep( TickType_t xExpectedIdleTime ) {
         counter_stop += 0x10000;
       }
       uint32_t counter_elapsed = counter_stop - counter_start;
-      uint32_t cycles_elapsed = (counter_elapsed * RTC_TICKS_HZ) / 9000;
 
-      s_analytics_device_sleep_cpu_cycles += cycles_elapsed;
-      s_analytics_app_sleep_cpu_cycles += cycles_elapsed;
+      // Calculate elapsed ticks and step FreeRTOS tick count
+      // LPTIM runs at ~9000 Hz, RTC_TICKS_HZ is 8 Hz
+      uint32_t ticks_elapsed = (counter_elapsed * RTC_TICKS_HZ) / 9000;
```

Comment:
the 9000hz is not true (far from true in some devices), we now have the real frequency measured agasinst HXT48, maybe add an internal API to retrieve it.